board_width: 5
board_height: 8
square_size: 0.025
